### PR TITLE
Downloading provided and test dependencies too

### PR DESCRIPTION
**Pull request by alexkubica**
Forked the original mvn-dd and modified to download dependencies with other scopes too.

### CODE REVIEW

**Comment by Schniz** [comment]
Cool :star2:

File: README.md [hunk 1]
```diff
@@ -24,4 +24,6 @@ and there will be a new `tar` file.
 
 that's it.
 
+**If an exception is thrown complaining about missing dependencies, you might need to add extra repositores to your directory, see `extra-repos.json` for example.**
```
Comment (Schniz) [inline README.md:27]: Is there a reason for it to happen?

Comment (alexkubica) [inline README.md:27]: Yeah, the jar searches on maven central repository but sometimes there are dependencies on other unknown repositories so it's necessary to add them specifically.

Comment (Schniz) [inline README.md:27]: :+1:

**Pull request merged**